src/olympia/users/tests/test_commands.py

```diff
@@ -52,6 +52,7 @@ def test_adds_supergroup(self):
             username='myusername',
             email='me@mozilla.org',
             add_to_supercreate_group=True,
+            fxa_id=000000,
```

Comment:
`fxa_id` is a UUID4 so it's a CharField that is exactly 32 characters long. We don't verify it in the database but we should use proper values in tests if possible.

```python
import uuid
uuid.uuid4().hex
```

should give you some random value.